.rubocop.yml

```diff
@@ -119,19 +121,87 @@ Naming/VariableNumber: # TODO enable & check offenses, default or snake_case?
 Naming/MethodParameterName:
   Enabled: false
 
-Style/ArgumentsForwarding:
+Performance/AncestorsInclude:
+  Enabled: false
+
+Performance/BigDecimalWithNumericArgument:
+  Enabled: true
+
+Performance/BlockGivenWithExplicitBlock:
   Enabled: true
 
-Style/CaseLikeIf: # TODO enable & check offenses
+# https://github.com/rubocop/rubocop-performance/issues/99#issuecomment-582560228
+Performance/Caller:
   Enabled: false
 
+# code readibility trumps performance improvement
+Performance/Casecmp:
+  Enabled: false
+
+# would add many lines, but does not improve performance significantly
+Performance/ChainArrayAllocation:
+  Enabled: false
+
+# code readibility trumps performance improvement
+Performance/CollectionLiteralInLoop:
+  Enabled: false
+
```

Comment:
Any reason why [CompareWithBlock](https://docs.rubocop.org/rubocop-performance/cops_performance.html#performancecomparewithblock) is not included here? That page says it's enabled by default, so does that mean that loading in `rubocop-performance` is enough to turn this cop on?

Reply:
OK I see it is enabled without having to specify it here 👍